src/screens/ProfileScreen/index.js

```diff
@@ -98,20 +98,12 @@ const Header = (props) => {
         headerHeightRef.current = headerHeightLayout;
       }}>
       <View style={styles.content}>
-        <View style={{flexDirection: 'row', alignContent: 'center', alignItems: 'center'}}>
+        <View style={{flexDirection: 'row'}}>
           <ProfilePicture
             onImageContainerClick={changeImage}
             profilePicPath={dataMain.profile_pic_path}
-            karmaScore={Math.floor(dataMain.karma_score)}
           />
-          <View
-            style={{
-              flexDirection: 'column',
-              paddingHorizontal: 14,
-              paddingVertical: 5,
-              justifyContent: 'center'
-            }}>
-            <KarmaScore score={Math.floor(dataMain.karma_score)} />
+          <View style={{marginLeft: 20}}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The layout and styling of the profile picture and follow info row have been changed. Make sure these changes don't break the UI or UX.
```diff
-        <View style={{flexDirection: 'row', alignContent: 'center', alignItems: 'center'}}>
+        <View style={{flexDirection: 'row'}}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->